### PR TITLE
Update autoconfiguration.md - Round 2

### PR DESCRIPTION
**Pull request by kbond**

### CODE REVIEW

**Comment by kbond** [comment]
Regarding `AppExtensionRuntime`: 

- I think this is a fairly new thing the maker bundle creates.
- It isn't strictly required (adding this logic directly in the extension would work fine)
- Should I create a little aside to describe why it's needed? Since the maker bundle creates this for us, I feel like we should. I also recently saw on the Symfony Slack someone confused as to why it's needed.

**Comment by bocharsky-bw** [comment]
> Should I create a little aside to describe why it's needed? Since the maker bundle creates this for us, I feel like we should. I also recently saw on the Symfony Slack someone confused as to why it's needed.

Yes, please! Sorry, sometimes I'm bad at describing such asides, or spell in advance what we are going to do and why... it may feel like I'm just start doing it 🙈  Ryan usually do a great job on such things, so if you see that it would be great to have some asides. - definitely do it, please 👍

File: sfcasts/autoconfiguration.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+This real-time ISS location feature is *cool*, but it would be *even cooler* if we could see this on *every* page, not just the homepage. How can we do that? We *could* pass the data in every action, *but* that's not ideal. Instead, we're going to create a custom Twig function that will fetch the *actual* data in the template. That way, we can render ISS location data in our `base.html.twig` file without passing it from every controller. Sound good? Let's get started!
 
 First, we need to create a Twig extension. In a previous course, we installed Symfony Maker Bundle. Let's see if that can help generate some boilerplate code. At your terminal, run:
 
 ```terminal
 bin/console make:
 ```
 
-This shows us list of all the commands at our disposal that *make* something, and... check it out! We have one that's Twig related - `make:twig-extension`. That's what we're looking for! Run that:
+and hit enter. We get an error but this shows us a list of all the commands at our disposal that *make* something, and... check it out! We have one that's Twig related - `make:twig-extension`. That's what we're looking for! Run that:
```
Comment (bocharsky-bw) [inline sfcasts/autoconfiguration.md:11]: That's what I intended to say in my head 👍

File: sfcasts/autoconfiguration.md [hunk 2]
```diff
@@ -20,29 +20,31 @@ Over here, we can see that a method is called on `AppExtensionRuntime::class`. R
...
+By the way, if you're curious on what this separate `AppExtensionRuntime` is for, great eye! *Extension runtimes* have always been in Twig but only recently have they been promoted - mostly thanks to the maker bundle. We could inject the services directly in our Twig extension and house all the logic there, but this comes with a disadvantage: since Twig extensions are loaded whenever Twig is used, the extension, and all its dependencies are also loaded. Even when not using a particular extension's function or filter. Twig extension runtimes are a way to make the extension logic *lazy*. The runtime service is only instantiated when and if its needed. In our example here, it isn't really helping us as we're showing the ISS location data on every page but you can imagine a function or filter that's only used on a few pages in your app. It's a best practice to keep your Twig extensions as light-weight as possible with none, or very few dependencies and push all the heavy-lifting to extension runtimes.
```
Comment (bocharsky-bw) [inline sfcasts/autoconfiguration.md:45]: Awesome, thanks for the additional explanation 👍

**Review by bocharsky-bw — APPROVED** [review]

**Pull request merged**